### PR TITLE
Changes for OpenShift v4 / oidc

### PR DESCRIPTION
**Pull request by ozRunAI**

### CODE REVIEW

File: docs/admin/runai-setup/authentication/sso.md [hunk 1]
```diff
@@ -26,35 +26,12 @@ For each of the SSO options, there are prerequisites that should be considered.
...
-2. Run the following command to apply the OAuthClient object to the environment. Create the object on OpenShift cluster where you define your OpenShift IDP:
-```
-oc apply <file name>
-```
-3. Check that the file has been applied successfully by running the following command:
-```
-oc get oauthclient
-```
+* **OpenShift OAuth client** - see [Registering an additional OAuth client](https://docs.openshift.com/container-platform/4.16/authentication/configuring-oauth-clients.html#oauth-register-additional-client_configuring-oauth-clients){target=_blank}
+* **Base URL**&mdash; - the OpenShift API Server endpoint (example: https://api.<cluster-url>:6443)
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/sso.md:32]: @ozRunAI no need for a "-" when useing the mdash code. "-" is an "en-dash". 
```suggestion
* **Base URL**&mdash;the OpenShift API Server endpoint (example: https://api.<cluster-url>:6443)
```

File: docs/admin/runai-setup/authentication/sso.md [hunk 2]
```diff
@@ -26,35 +26,12 @@ For each of the SSO options, there are prerequisites that should be considered.
...
+* **ClientID**&mdash;  - the ID used to identify the client with the Authorization Server.
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/sso.md:33]:
```suggestion
* **ClientID**&mdash;the ID used to identify the client with the Authorization Server.
```

File: docs/admin/runai-setup/authentication/sso.md [hunk 3]
```diff
@@ -106,9 +83,10 @@ You can configure your IdP to map several IdP attributes:
         1. In the `Discovery URL` field, enter the discovery URL .
         2. In the `Client ID` field, enter the client ID.
         3. In the `Client Secret` field, enter the client secret.
-	4. Add the OIDC scope to be used during authentication to authorize access to a user's details (optional).  Each scope returns a set of user attributes.  The scope must match the names in your identity provider.
-        5. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
-        6.When complete, press `Save`.
+		4. Copy the `Redirect URL` and use it in your identity provider.
+		5. Add the OIDC scope to be used during authentication to authorize access to a user's details (optional).  Each scope returns a set of user attributes.  The scope must match the names in your identity provider.
+        6. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
+        7.When complete, press `Save`.
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/sso.md:89]:
```suggestion
	4. Copy the `Redirect URL` and use it in your identity provider.
	5. Add the OIDC scope to be used during authentication to authorize access to a user's details (optional).  Each scope returns a set of user attributes.  The scope must match the names in your identity provider.
        6. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
        7.When complete, press `Save`.
```
@ozRunAI Please be careful of the spaces you add. If these are meant to be indented ordered list, then you need to add spaces in the lines above before you indent them.

File: docs/admin/runai-setup/authentication/sso.md [hunk 4]
```diff
@@ -117,16 +95,25 @@ You can configure your IdP to map several IdP attributes:
...
 	=== "OpenShift V4"
 
-        1. In the `Base URL` field, enter the OpenShift Base URL (https://api.<your-openshift-domain>:6443).
+        1. In the `Base URL` field, enter the OpenShift API server endpoint (https://api.<your-openshift-domain>:6443).
         2. In the `Client ID` field, enter the client ID.
         3. In the `Client Secret` field, enter the client secret.
-	4. Add the OIDC scope to be used during authentication to authorize access to a user's details (optional).  Each scope returns a set of user attributes.  The scope must match the names in your identity provider.
+		4. Add the OIDC scope to be used during authentication to authorize access to a user's details (optional).  Each scope returns a set of user attributes.  The scope must match the names in your identity provider.
         5. In the `User attributes` field enter the attribute and the value in the identity provider (optional).
         6. When complete, press `Save`.
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/authentication/sso.md:107]:
```suggestion
	4. Add the OIDC scope to be used during authentication to authorize access to a user's details (optional). Each scope returns a set of user attributes.  The scope must match the names in your identity provider.
        5. In the `User attributes` field enter the attribute and the value in the identity provider (optional).
        6. When complete, press `Save`.
```

**Comment by ozRunAI** [comment]
@jasonnovichRunAI please do not merge until @talevy-runai approves the PR

**Review by jasonnovichRunAI — COMMENTED** [review]
@ozRunAI I made some changes - please review so that for the future the markdown will look right. 8-)

**Review by talevy-runai — APPROVED** [review]

**Review by jasonnovichRunAI — APPROVED** [review]

**Pull request merged**